pkgs/development/python-modules/ludios_wpull/default.nix

```diff
@@ -0,0 +1,68 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchPypi
+, chardet
+, dnspython
+, html5-parser
+, lxml
+, namedlist
+, psutil
+, setuptools
+, sqlalchemy
+, tornado
+, yapsy
+, python
+, pythonOlder
+# Override Python packages using
+# self: super: { pkg = super.pkg.overridePythonAttrs (oldAttrs: { ... }); }
+# Applied after defaultOverrides
+, packageOverrides ? self: super: {}
+}:
+
+let
+  defaultOverrides = [
+    # override the version of packages pinned in pyproject.toml
+    (self: super: {
+      sqlalchemy = super.sqlalchemy.overridePythonAttrs (oldAttrs: rec {
```

Comment:
overrides can not be done for python modules, because it would introduce clashes with other dependencies

overrides can be done for applications. from my initial look wpull is intended to be used as an application and not python module ( eg import )

if that's the case then it needs to be moved to by-name https://github.com/NixOS/nixpkgs/blob/master/pkgs/by-name/README.md

Reply:
Moved, thanks. And thank for your patience.